servers/rest-services/src/main/java/org/projectnessie/services/rest/BackendLimitExceededExceptionMapper.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.services.rest;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.Provider;
+import org.projectnessie.error.ErrorCode;
+import org.projectnessie.services.rest.exceptions.BaseExceptionMapper;
+import org.projectnessie.services.rest.exceptions.NessieExceptionMapper;
+import org.projectnessie.services.rest.exceptions.ValidationExceptionMapper;
+import org.projectnessie.versioned.BackendLimitExceededException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * "Special" implementation for exceptions that extend {@link ConstraintViolationException}, as
```

Comment:
`BackendLimitExceededException` is not a sub-class of `ConstraintViolationException`